diff --git a/node_modules/vue-carousel/src/Carousel.vue b/node_modules/vue-carousel/src/Carousel.vue
index dd70b0a..1dd6835 100644
--- a/node_modules/vue-carousel/src/Carousel.vue
+++ b/node_modules/vue-carousel/src/Carousel.vue
@@ -868,11 +868,14 @@ export default {
       const remainderOffset =
         lastFullPageOffset +
         this.slideWidth * (this.slideCount % this.currentPerPage);
-      if (this.offset > (lastFullPageOffset + remainderOffset) / 2) {
-        this.offset = remainderOffset;
-      } else {
-        this.offset = width * Math.round(this.offset / width);
-      }
+      // if (this.offset > (lastFullPageOffset + remainderOffset) / 2) {
+      //   this.offset = remainderOffset;
+      // } else {
+      //   this.offset = width * Math.round(this.offset / width);
+      // }
+      
+      // Workaround for https://github.com/SSENSE/vue-carousel/issues/345
+      this.offset = width * Math.round(this.offset / width);
 
       // clamp the offset between 0 -> maxOffset
       this.offset = Math.max(0, Math.min(this.offset, this.maxOffset));
